Sprint-2/implement/querystring.test.js

```diff
@@ -3,10 +3,16 @@
 // Below is one test case for an edge case the implementation doesn't handle well.
 // Fix the implementation for this test, and try to think of as many other edge cases as possible - write tests and fix those too.
 
-const parseQueryString = require("./querystring.js")
+const parseQueryString = require("./querystring.js");
 
-test("parses querystring values containing =", () => {
-  expect(parseQueryString("equation=x=y+1")).toEqual({
-    "equation": "x=y+1",
+describe("querystring", () => {
+  test("parses querystring values containing =", () => {
+    expect(parseQueryString("equation=x=y+1")).toEqual({ equation: "x=y+1" });
   });
-});
+  test("parses multiple key-value pairs", () => {
+    expect(parseQueryString("color1=blue&color2=red")).toEqual({
+      color1: "blue",
+      color2: "red",
+    });
+  });
+});
```

Comment:
Your function is pretty solid. Why not test also the following "abnormal" query strings?
```
parseQueryString("a=b&=&c=d")
parseQueryString("a=")
parseQueryString("=b")
parseQueryString("a=b&&c=d")
parseQueryString("a%25b=c%26d`)
```
Note: the `%25` and `%26` are ***URL encoded*** or ***percent encoded*** characters.